v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/constants/Constants.java

```diff
@@ -80,4 +80,8 @@ public class Constants {
   // Message written to the file for filtered records
   public static final String FILTERED_TAG_MESSAGE =
       "Filtered record from custom transformation in reverse replication";
+
+  // Sentinel Shard IDs for error handling
+  public static final String RETRYABLE_SENTINEL_SHARD_ID = "retryable_error_shard";
```

Comment:
nit: We don't really use sentinel in this context in the code much.

Maybe just use a more obvious RETRYABLE_ERROR_SHARD_ID and SEVERE_ERROR_SHARD_ID